src/compat/eme/custom_media_keys/ie11_media_keys.ts

```diff
@@ -146,9 +146,17 @@ export default function getIE11MediaKeysCallbacks() {
     return MSMediaKeysConstructor.isTypeSupported(keySystem);
   };
   const createCustomMediaKeys = (keyType: string) => new IE11CustomMediaKeys(keyType);
+  const customSetMediaKeys = (elt: HTMLMediaElement,
+                              mediaKeys: ICustomMediaKeys|null): void => {
+    if (mediaKeys === null) {
```

Comment:
Considering IE11's previous implementation:
```
    this._videoElement = videoElement;
    /* tslint:disable no-unsafe-any */
    if ((this._videoElement as any).msSetMediaKeys !== undefined) {
      return (this._videoElement as any).msSetMediaKeys(this._mediaKeys);
    }
    /* tslint:enable no-unsafe-any */
```
Maybe we could still do something like `mediaElement.msSetMediaKeys(null);`

Reply:
Actually, mediaElement.msSetMediaKeys(null) throws an error.

Reply:
OK

Reply:
It might make sense to document that